app/assets/stylesheets/application.scss

```diff
@@ -49,6 +49,26 @@ body > nav {
   }
 }
 
+.site-admins {
+  header {
+    margin: 1rem 0;
+  }
+
+  img {
```

Comment:
One of our admins has a very small image (no large version at all).  Can you add some CSS to ensure the card/grid stays consistent? See attached image.

<img width="508" alt="small-avatar-breaks-grid" src="https://user-images.githubusercontent.com/16705/52525003-0e4a3800-2c58-11e9-8184-39765dd0454b.png">